src/components/CommentItem.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comment: Comment,
+  onCommentDelete: (id: number) => void,
+};
+
+export const CommentItem: React.FC<Props> = ({ comment, onCommentDelete }) => (
+  <article className="message is-small" data-cy="Comment">
+    <div className="message-header">
+      <a href={`mailto:${comment.email}`} data-cy="CommentAuthor">
```

Comment:
Let's do destructuring for the comment

Reply:
Hi, I can't run these tests. I tried to check manualy - seems all should be fine.